content/includes/engine-install-methods-apt.md

```diff
@@ -0,0 +1,13 @@
+You can install Docker Engine in different ways, depending on your needs:
+
+- Docker Engine comes bundled with
+  [Docker Desktop for Linux](../../desktop/install/linux-install.md). This is
+  the easiest and quickest way to get started.
+
+- Set up and install Docker Engine from
+  [Docker's Apt repository](#install-using-the-repository).
```

Comment:
Note for reviewer: We inconsistently say `apt` and Apt in our docs, and I've seen it multiple ways in other docs. Thoughts on how we want to standardize this here?

Reply:
I've noticed this as well and deferred making my mind up 🙈 This is an acronym for Advanced Packaging Tool, so technically we should use APT. But you only really care about this tool as a CLI invocation, much like `curl`. So IMO, we're better off using `apt` for this (and similar utilities) - let's do that

Reply:
Apt is definitely wrong and my bad - not sure what I was thinking

Reply:
K will update to use `apt` 👍